dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -273,8 +248,36 @@
                         </div>
                       </div>
                     </FormKit>
-                  </FormKit>
-                </div>
+
+                    <FormKit :name="financialServiceOption.value" type="group">
```

Comment:
Why is this not the same formkit group as its upper sibling. They have the same name.